cell_based/src/mesh/PottsMeshGenerator.hpp

```diff
@@ -91,10 +92,12 @@ class PottsMeshGenerator
                        bool isPeriodicInZ = false);
 
     /**
-     * Destructor - deletes the mesh object and pointer.
```

Comment:
@fcooper8472 might suggest these should use the `= default` rather than empty user-defined destructors (I think this can occasionally even save the compiler from making a destructor if there is no work to do, but haven't fully got my head around it!)

Reply:
Thanks @mirams. That seems like good practice according to the [core guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all):
>**Example, good** When a destructor needs to be declared just to make it virtual, it can be defined as defaulted.